tests/python/gaia-ui-tests/gaiatest/apps/settings/app.py

```diff
@@ -236,6 +264,10 @@ def open_browsing_privacy_settings(self):
         self._tap_menu_item(self._browsing_privacy_item_locator)
         return BrowsingPrivacy(self.marionette)
 
+    # method to open menu item that does not return object
+    def simple_open_menu(self, menu_locator):
```

Comment:
will incorporate.  only thing is that we need extra parameter for filename and class name, since 'DeviceInfo' class sits in 'device_info.py' file.

Also need to take care of cases where we don't have the classes as well